source/index.html.md

```diff
@@ -540,26 +547,115 @@ Exactly one payment method must be used.
   </dd>
+
   <dt>currency
     <span class="type">[A-Z]{3}</span>
   </dt>
   <dd>
     <a target="_blank" href="currencies.txt">3-letter currency code</a>. (Some exponents differ from ISO 4217.)
   </dd>
+
+  <dt>credential_on_file
+    <span class="type">(<code>store</code>|<code>use</code>)</span>
+  </dt>
+  <dd>
+    Indicate if the payment credential (e.g. PAN and expiry) will be stored for
+    future use where the payment credential is not provided by the
+    cardholder but collected from (encrypted) storage.
+    <br />
+    <code>store</code>: The payment credential will be stored; it may only be
+    stored if the authorization is approved.
+    <br />
+    <code>use</code>: The payment credential has already been stored and is now
+    being used.
+    <br />
+    Default:
+    <ul>
+      <li><code>use</code>, if <code>initiator</code> is <code>merchant</code>
+      (<code>store</code> is invalid),</li>
+      <li><code>store</code>, if the authorization is first-in-series.</li>
+    </ul>
+
+    <div class="type">Optional</code>
+  </dd>
+
+  <dt>initiator
+    <span class="type">(<code>cardholder</code>|<code>merchant</code>)</span>
+  </dt>
+  <dd>
+    The initiator of the authorization. An authorization is initiated by the
+    cardholder if the cardholder decided the transaction should be created.
+    This is regardless of whether a stored payment credential is being used.
+    <br />
+    For compliance reasons there should be a previous approved transaction (for
+    the cardholder and the merchant) marked with <code>storing</code> before
+    <code>initiator</code> may be <code>merchant</code>.
+    <br />
+    Default:
+    <ul>
+      <li><code>merchant</code>, if the authorization is
+      subsequent-in-subsequent (<code>cardholder</code> is invalid),</li>
+      <li><code>cardholder</code>, otherwise.</li>
+    </ul>
+
+    <div class="type">Optional</div>
+  </dd>
+
   <dt>ip
     <span class="type">[0-9.a-fA-F:]{3,45}</span>
   </dt>
   <dd>
     Cardholder's IP address. It must be a valid v4 or v6 address.
     <div class="type">Optional<br></div>
   </dd>
-  <dt>recurring
-    <span class="type">(true|false)</span>
+
+  <dt>reference
+    <span class="type">[\x20-\x7E]{1,30}
+      <a target="_blank" href= "http://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">
+        ASCII printable characters
+      </a>
+    </span>
   </dt>
   <dd>
-    Must be <code>true</code> for recurring payments.
+    A reference to an external object, such as an order number.
     <div class="type">Optional</div>
   </dd>
+
+  <dt>
+    series[type]
+    <span class="type"><code>recurring</code></span>
+  </dt>
+  <dd>
+    Indicate the type of series.
+    <br />
+    <code>recurring</code>: A series of transactions where the cardholder has
+    explicitly agreed that the merchant will repeatedly charge the cardholder at
+    regular, predetermined intervals that may not exceed 1 year.
```

Comment:
I've also read this above. Perhaps remove the redundancy.

Reply:
The part about not exceeding 1 year.

Reply:
(in line 341)

Reply:
Fixed in ada4b57.